vulns/CVE-2024-7646.json

```
@@ -0,0 +1,44 @@
+{
+  "id": "CVE-2024-7646",
+  "modified": "2024-08-16T16:10:31Z",
+  "published": "2024-08-16T16:10:31Z",
+  "summary": "Ingress-nginx Annotation Validation Bypass",
+  "details": "A security issue was discovered in ingress-nginx where an actor with permission to create Ingress objects (in the `networking.k8s.io` or `extensions` API group) can bypass annotation validation to inject arbitrary commands and obtain the credentials of the ingress-nginx controller. In the default configuration, that credential has access to all secrets in the cluster.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "/ingress-nginx"
```

cf. https://github.com/kubernetes/ingress-nginx/blob/eee2760907f029b808d95cd90abf290a9f0f09dd/go.mod#L1

```suggestion
        "name": "k8s.io/ingress-nginx"
```